EmbeddedResourceVirtualPathProvider/EmbeddedResourceVirtualDirectory.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Web.Hosting;
+
+namespace EmbeddedResourceVirtualPathProvider
+{
+	class EmbeddedResourceVirtualDirectory : VirtualDirectory
+	{
+		Dictionary<string, List<EmbeddedResource>> _resources;
+		Func<EmbeddedResource, EmbeddedResourceCacheControl> _cacheControl;
+
+		public EmbeddedResourceVirtualDirectory(string virtualPath, Dictionary<string, List<EmbeddedResource>> resources, Func<EmbeddedResource, EmbeddedResourceCacheControl> cacheControl) : base(virtualPath)
+		{
+			_resources = resources;
+			_cacheControl = cacheControl;
+		}
+
+		public override IEnumerable Children
+		{
+			get
+			{
+				return Files;
+			}
```

Comment:
Are subdirectories unnecessary for ASP.NET bundling, or will embedded resource bundling only work with single subdirectories? 

Sorry for dumb questions but I don't use Bundles that much, and if there's any unexpected behaviour I'll need to document it to reduce the number of people raising Github Issues.